openapi/servicenow/client.bal

```diff
@@ -0,0 +1,477 @@
+    # + return - If successful, returns a JSON object. Otherwise returns the relevant error.
+    remote isolated function getImportSetTable(string stagingTableName, string sys_id, string? accept = ()) returns json|error {
+        string  path = string `/api/now/import/${stagingTableName}/${sys_id}`;
+        map<any> headerValues = {"Accept": accept};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        json response = check self.clientEp-> get(path, accHeaders, targetType = json);
+        return response;
+    }
+    # Inserts incoming data into a specified staging table and triggers transformation based on predefined transform maps in the import set table.
+    #
+    # + stagingTableName - Name of the table from which to import the data.
+    # + payload - Name-value pairs to insert in the import fields in JSON format.
+    # + accept - Data format of the response body.
+    # + contentType - Data format of the request body.
+    # + return - If successful, returns a JSON object. Otherwise returns the relevant error.
+    remote isolated function insertImportSetTable(string stagingTableName, json payload, string? accept = (), string? contentType = ()) returns json|error {
+        string  path = string `/api/now/import/${stagingTableName}`;
+        map<any> headerValues = {"Accept": accept, "Content-Type": contentType};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        http:Request request = new;
+        request.setPayload(payload);
+        json response = check self.clientEp->post(path, request, headers = accHeaders, targetType=json);
+        return response;
+    }
+    # Inserts multiple records into a specified staging table and triggers transformation based on predefined transform maps or Robust Transform Engine (RTE) configurations in a single request.
+    #
+    # + stagingTableName - Name of the table from which to import the data.
+    # + payload - Name-value pairs to insert in the import fields in JSON format.
+    # + multiImportSetId - Sys_id of an entry in the Multi Import Sets [sys_multi_import_set] table. If specified, adds the current import to this multiple import set instead of adding to a new multiple import set.
+    # + runAfter - Sys_id of an entry in the Import Sets [sys_import_set] table. Enables running the current import set after the specified import set is complete. You can use this parameter to enforce the sequential order of imports. This parameter is only valid in asynchronous transformations.
+    # + accept - Data format of the response body.
+    # + contentType - Data format of the request body.
+    # + return - If successful, returns a JSON object. Otherwise returns the relevant error.
+    remote isolated function insertMultipleImportSetTable(string stagingTableName, json payload, string? multiImportSetId = (), string? runAfter = (), string? accept = (), string? contentType = ()) returns json|error {
```

Comment:
The function name must be `createMultipleImportSet`

Reply:
Fixed by https://github.com/kasthuriraajan/ballerinax-openapi-connectors/commit/7d722e1656dd4084768a6a6f0aab1753b97a5d98